migration/v2.18.0-ck8s1-v2.19.x-ck8s1/upgrade-cluster.md

```diff
@@ -15,3 +15,11 @@
 1. Upgrade your cluster by running `./bin/ck8s-kubespray run-playbook sc upgrade-cluster.yml -b`.
 
 1. Upgrade your cluster by running `./bin/ck8s-kubespray run-playbook wc upgrade-cluster.yml -b`.
+
+1. After the upgrade, if you want to start using containerd follow [this guide](https://kubespray.io/#/docs/upgrades/migrate_docker2containerd) to migrate or set `container_manager: containerd` to `${CK8S_CONFIG_PATH}/{wc,sc}-config/group_vars/k8s_cluster/ck8s-k8s-cluster.yaml` and run
+
+```
+./bin/ck8s-kubespray run-playbook {sc|wc} ../migration/v2.18.0-ck8s1-v2.19.x-ck8s1/migrate-to-containerd.yml -b
+```
+
+**NOTE**: If you are running [Compliant Kubernetes Apps](https://github.com/elastisys/compliantkubernetes-apps). Make sure you're running a version that has support for conatinerd.
```

Comment:
Do we need to have this for the 2.19 upgrade as well? (I guess it does not hurt to have it here, but I had assumed that we will upgrade all clusters while on 2.18).

Reply:
Never mind, let's have this in 2.19 as well.

Reply:
I guess we can have it in both, as it doesn't hurt.